examples/llms/instrumentation.ts

```diff
@@ -0,0 +1,27 @@
+//////////////////////////////////////////////////////////////////////////////////////////////////
+/////// RUN THIS EXAMPLE VIA `yarn start:telemetry ./examples/llms/instrumentation.ts` ///////////
+//////////////////////////////////////////////////////////////////////////////////////////////////
+import { BaseMessage, Role } from "bee-agent-framework/llms/primitives/message";
+import { OllamaChatLLM } from "bee-agent-framework/adapters/ollama/chat";
+import { Logger } from "bee-agent-framework/logger/logger";
+
+Logger.root.level = "silent"; // disable internal logs
+const logger = new Logger({ name: "app", level: "trace" });
+
+const llm = new OllamaChatLLM({
+  modelId: "llama3.1", // llama3.1:70b for better performance
+});
+
+const response = await llm.generate([
+  BaseMessage.of({
+    role: Role.USER,
+    text: "hello",
+  }),
+]);
+
+logger.info(`LLM 🤖 (txt) : ${response.getTextContent()}`);
+
+// Wait briefly to ensure all telemetry data has been processed
+setTimeout(() => {
+  logger.info("Process exiting after OpenTelemetry flush.");
+}, 5_000); // Adjust the delay as needed
```

Comment:
I believe that the process should not close if the even loop is non-empty (something is waiting for a dispatch).

Reply:
no, the process finishes without this script. I could not find the way, how to wait for traces.

Reply:
You cannot force the spans. I tried:
```
provider.forceFlush()
```
but this function isn't available for current versions.
and the `unref` does not exist.

This setting:
```
 scheduledDelayMillis: 0,
  exportTimeoutMillis: 0
  ```
  does not help either.

Reply:
So, let me clarify. If you don't set the timeout, the application closes, and no records are sent.

Reply:
exactly